.travis.yml

```diff
@@ -1,14 +1,15 @@
 sudo: false
 language: go
 go:
-  - tip
+  - master
```

Comment:
`tip` is a legacy reference coming from the days when the Go project used mercurial instead of git